### PR TITLE
Update the Product Alerts page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request:
- Updates the product alert image;
- Adds the Alert Email Sender options;
- Replaces "Sign up for price alert", "Sign up to get notified when this product is back in stock" with "Notify me when the price drops", "Notify me when this product is in stock".

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/inventory-product-alerts.html

### CODE REVIEW

File: src/catalog/inventory-product-alerts.md [hunk 1]
```diff
@@ -36,12 +36,17 @@ _List of Product and Price Alert Subscriptions_
     - To offer alerts when out-of-stock products become available again, set **Allow Alert When Product Comes Back in Stock** to `Yes`.
 
       {:.bs-callout-info}
-      The _Sign up to get notified when this product is back in stock_ message appears only when **Display Out of Stock Products** is set to `Yes` (located in the Configuration at Catalog > Inventory > Stock Options).
+      The _Notify me when this product is in stock_ message appears only when **Display Out of Stock Products** is set to `Yes` (located in the Configuration at Catalog > Inventory > Stock Options).
 
     - Set **Stock Alert Email Template** to the template that you want to use for product stock alerts.
 
     - Set **Alert Email Sender** to the [store contact]({% link stores/store-email-addresses.md %}) that you want to appear as the sender of the email alert.
```
Comment (erikmarr) [inline src/catalog/inventory-product-alerts.md:43]:
```suggestion
    - Set **Alert Email Sender** to the [store contact]({% link stores/store-email-addresses.md %}) that you want to appear as the sender of the email alert. Learn more about [store email addresses]({% link  configuration/general/store-email-addresses.md %})
```

File: src/catalog/inventory-product-alerts.md [hunk 2]
```diff
@@ -36,12 +36,17 @@ _List of Product and Price Alert Subscriptions_
...
 
+      |General Contact|The email address associated with the General Contact identity. |
```
Comment (erikmarr) [inline src/catalog/inventory-product-alerts.md:45]: I think we should delete this list as the link I suggested above explains these options in more detail.

**Review by erikmarr — CHANGES_REQUESTED** [review]

File: src/catalog/inventory-product-alerts.md (hunk 2 above)
Comment (ZhovnirMaria) [reply src/catalog/inventory-product-alerts.md]: @erikmarr 
Hi, thank you! Done✅ 
Please, approve.

File: src/catalog/inventory-product-alerts.md [hunk 3]
```diff
@@ -36,11 +36,11 @@ _List of Product and Price Alert Subscriptions_
     - To offer alerts when out-of-stock products become available again, set **Allow Alert When Product Comes Back in Stock** to `Yes`.
 
       {:.bs-callout-info}
-      The _Sign up to get notified when this product is back in stock_ message appears only when **Display Out of Stock Products** is set to `Yes` (located in the Configuration at Catalog > Inventory > Stock Options).
+      The _Notify me when this product is in stock_ message appears only when **Display Out of Stock Products** is set to `Yes` (located in the Configuration at Catalog > Inventory > Stock Options).
 
     - Set **Stock Alert Email Template** to the template that you want to use for product stock alerts.
 
-    - Set **Alert Email Sender** to the [store contact]({% link stores/store-email-addresses.md %}) that you want to appear as the sender of the email alert.
+    - Set **Alert Email Sender** to the [store contact]({% link stores/store-email-addresses.md %}) that you want to appear as the sender of the email alert. Learn more about [store email addresses]({% link  configuration/general/store-email-addresses.md %})
```
Comment (erikmarr) [inline src/catalog/inventory-product-alerts.md:43]:
```suggestion
    - Set **Alert Email Sender** to the [store contact]({% link stores/store-email-addresses.md %}) that you want to appear as the sender of the email alert. Learn more about [store email addresses]({% link configuration/general/store-email-addresses.md %})
```

**Review by erikmarr — APPROVED** [review]

**Comment by erikmarr** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@erikmarr the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1216`.

**Pull request merged**